<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextAnalyzerClass</class>
 <widget class="QMainWindow" name="TextAnalyzerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextAnalyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="textTab">
     <attribute name="title">
      <string>텍스트</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>411</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="doAnalyze">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>분 석</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resultTab">
     <attribute name="title">
      <string>결과</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>551</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open"/>
    <addaction name="save"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open"/>
   <addaction name="save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="textanalyzer.qrc"/>
 </resources>
 <connections/>
</ui>
